demo/psa/manual-end-to-end.md

```diff
@@ -5,13 +5,58 @@ end to end PSA Demonstration.
 
 ## Preconditions
 
-* One need to install `jq` and `curl`.
+* One need to install `GO`, `jq`,`sqilte3`,`tmux` and `curl`.
+
+Installing Go, use the below mentioned path to install go in your system.
+
+`https://go.dev/doc/install`
+
+installing jq.
+```sh
+sudo apt update||sudo apt install jq||jq --version
+```
+
+Installing sqlite3.
+```sh
+sudo apt update
+sudo apt install sqlite3 
+sqlite3 --version
+```
+You will get an output like this:
+`3.31.1 2020-01-27 19:55:54 3bfa9cc97da10598521b342961df8f5f68c7388fa117345eeb516eaa837balt1`
+
+Installing Curl.
+```sh
+sudo apt-get install autoconf automake libtool curl make g++ unzip
```

Comment:
not an Ubuntu user, but I am a bit puzzled by the huge list of co-packages here.

I'd have thought a simple `sudo apt install curl` would suffice?